src/2_RTB/NormeDiProgetto/content.typ

```diff
@@ -17,64 +17,91 @@ Il presente documento ha come scopo la definizione delle #glossary[best practice
 
 La modifica di un documento avviene allo stesso modo, saltando il passo 2.
 Ogni cambiamento di stato è accompagnato dal conseguente movimento della issue, associata allo sviluppo, attraverso le diverse corsie dell'issue tracking system.
-La revisione del documento avviene tramite apposito metodo nell'ITS, attraverso la revisione nella _pull request_, indicando i punti in cui si richiede la modifica, il motivo della richiesta e una proposta se necessario. Anche in questo caso il revisore si occupa di spostare la issue di riferimento, nelle corsie appropriate, chiudendole se la revisione è terminata con successo, o spostandola nuovamente nella corsia _In Progress_ quando vengono richieste altre modifiche.
 
-=== Nomenclatura
-La consueta nomeclatura per i documenti si ottiene unendo, attraverso un underscore (`_`), il nome del file in _CamelCase_ senza spazi (`NomeDelFile`) e la sua versione (`3.5`). Ad esempio `NormeDiProgetto_2.6.pdf`.
-Nel caso di documenti il cui nome contiene una data, essa si inserisce dopo il nome, ma prima della versione, sempre separandolo con gli underscores, nella forma `ggmmaa` senza separatori tra i singoli componenti della data: `gg` rappresenta il giorno, sempre scritto in due cifre, allo stesso modo `mm` rappresenta il mese, mentre l'anno è rappresentato da `aa`, che corrisponde alle ultime due cifre dell'anno corrente.
+== Gestione della Configurazione
 
-=== Versionamento <versionamento>
-Il versionamento avviene secondo il seguente formato *x.y*:
-- *y* si incrementa una volta effettuata una modifica e la sua conseguente revisione;
-- *x* si incrementa quando si effettua la modifica definitiva in vista di una revisione di avanzamento, questo comporta l'azzeramento di *y*.
+=== Issue Tracking System <its>
+Come ITS si utilizza _Github_, attraverso le funzioni di _Project_, _Issues_ e _Pull requests_, che garantisce una struttura all'organizzazione di progetto.
+
+Le #glossary[Corsie di Stato] descrivono lo stato attuale delle attività, all'interno del _Project_ nell'_ITS_ sono presenti:
+/ Backlog: attività individuate da svolgere.
+/ Ready: attività individuate per il completamento durante il prossimo #glossary[sprint].
+/ In Progress: attività che sono in corso d'opera da parte dei redattori.
+/ Ready to Review: attività svolte che sono pronte per essere verificate.
+/ In Review: attività in corso di verifica da parte dei verificatori.
+/ Done: attività le cui modifiche sono state verificate e accettate.
+
+=== Strumento di Condivisione
+Per la condivisione veloce o la creazione di bozze si utilizza _Google Drive_. Uno dei suoi principali casi d'uso comprende la collaborazione in tempo reale a grandi parti di testo che poi andranno inserite all'interno della documentazione (ad esempio durante la creazione di un nuovo documento). Viene inoltre utilizzato come sistema per l'immagazzinamento di conoscenze acquisite durante il corso dello svolgimento del progetto.
```

Comment:
*durante lo svolgimento 
*uno dei suoi principali casi d'uso consiste nella collaborazione in tempo reale nella stesura di sezione testuali ampie,  da inserire successivamente nella documentazione. [Questo risulta particolarmente utile nel momento in cui il documento è alla sua prima stesura.]